tests/run_tests.py

```diff
@@ -1,248 +1,250 @@
-#!/usr/bin/env python
```

Comment:
I dont know what happened with that file.

Reply:
Maybe different endline